app/controllers/response_templates_controller.rb

```diff
@@ -31,9 +31,12 @@ def index
 
   def create
     authorize ResponseTemplate
-    response_template.user_id = current_user.id
+
+    unless current_user.has_trusted_role? && params[:response_template][:type_of] == "mod_comment"
```

Comment:
My gut didn't like that either -- feels like it only worked because it was new material, so there wasn't already a test-case here for the other user-types (admin or moderator) that aren't consistently guaranteed to be `trusted`. This gets a bit verbose in e51cf7871 but I think it might be worth the extra characters?